iphone/Classes/TiUIListView.m

```diff
@@ -1815,7 +1815,35 @@ - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView
         [self.proxy fireEvent:@"dragstart" withObject:nil withSource:self.proxy propagate:NO reportSuccess:NO errorCode:0 message:nil];
     }
 }
-
+-(void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset
+{
+    if ([[self proxy] _hasListeners:@"scrolling"]) {
+        NSString* direction = nil;
+        
+        if (velocity.y > 0) {
+            direction = @"up";
+        }
+        
+        if (velocity.y < 0) {
+            direction = @"down";
+        }
+        
+        NSMutableDictionary *event = [NSMutableDictionary dictionaryWithDictionary:@{
+                                                                                     @"targetContentOffset": @{
+                                                                                             @"x": NUMFLOAT(targetContentOffset->x),
+                                                                                             @"y": NUMFLOAT(targetContentOffset->y),
+                                                                                             },
+                                                                                     @"velocity": NUMFLOAT(velocity.y)
+                                                                                     }];
```

Comment:
The copy-and-paste caused formatting issues here, please refer to the gist and format manually.